api/apps/api/src/modules/geo-feature-tags/dto/tag-info.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNotEmpty, MaxLength, Validate } from 'class-validator';
+import { AppConfig } from '@marxan-api/utils/config.utils';
+import { IsValidTagNameValidator } from '@marxan-api/modules/geo-feature-tags/validators/is-valid-tag-name.custom.validator';
+
+const tagMaxlength = AppConfig.get<number>('marxan.maxTagLength');
+
+export class TagInfoDTO {
+  @ApiProperty()
+  @IsNotEmpty()
+  @Validate(IsValidTagNameValidator)
+  @MaxLength(tagMaxlength, {
+    message: `tagName length should not be longer than ${tagMaxlength} characters`,
```

Comment:
very tiny, mostly style - in case frontend would like to pass on an error message from the API, I suggest to make these as user-meaningful as possible (for example, "A tag should not be longer than ... characters").

I see no harm in trying to make errors/warnings as simple and meaningful as possible for end users even if they end up not being displayed. When triaging issues, the error details in responses should give us enough information to get to the implementation details without having to surface these through error messages.

Reply:
And yes, I know we don't have universally good examples across this application on this front - something for me to add to documentation on project conventions, I guess 😬